src/main/java/in/twizmwaz/cardinal/command/TeleportCommands.java

```diff
@@ -21,14 +22,15 @@
     @Command(aliases = {"tp", "teleport"}, desc = "Teleport players.", usage = "<player> [player], [player] [[~]x] [[~]y] [[~]z]", min = 1, max = 4)
     public static void teleport(final CommandContext cmd, CommandSender sender) throws CommandException {
         if (sender instanceof Player) {
+            Player SendingPlayer = (Player) sender;
```

Comment:
Never start variables like this with an uppercase letter. Perhaps just call this variable `player` and if `player` is already being used somewhere else, then change the other variables to `target` or something similar.